schemas/clab.schema.json

```diff
@@ -902,9 +907,30 @@
                         "vr-vsrx": {
                             "$ref": "#/definitions/node-config"
                         },
+                        "vr-vjunosrouter": {
+                            "$ref": "#/definitions/node-config"
+                        },
+                        "vr-juniper_vjunosrouter": {
+                            "$ref": "#/definitions/node-config"
+                        },
```

Comment:
when you remove the `vr-` kind names from the go code we should also remove it from the schema